easybuild/easyconfigs/g/GCCcore/GCCcore-10.2.0.eb

```
@@ -17,13 +17,17 @@ source_urls = [
     'ftp://gcc.gnu.org/pub/gcc/infrastructure/',  # GCC dependencies
     'http://gcc.cybermirror.org/infrastructure/',  # HTTP mirror for GCC dependencies
     'http://isl.gforge.inria.fr/',  # original HTTP source for ISL
+    'ftp://sourceware.org/pub/newlib/', # for newlib
+    'https://github.com/MentorEmbedded/nvptx-tools/archive', # for nvptx-tools
 ]
 sources = [
     'gcc-%(version)s.tar.gz',
     'gmp-6.2.0.tar.bz2',
     'mpfr-4.1.0.tar.bz2',
     'mpc-1.1.0.tar.gz',
     'isl-0.22.1.tar.bz2',
+    'newlib-3.3.0.tar.gz',
+    '5f6f343.tar.gz', # nvptx-tools
```

@bartoldeman This should be renamed on download:

```suggestion
    {'download_filename': '5f6f343.tar.gz', 'filename': 'nvptx-tools-20180301.tar.gz'},

```